### PR TITLE
Fix for standard mode: move additional files into proper place

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Move additional files during standard mode into proper place
* Change links to actual repo over the project
* Update save-cli version and it's API

### CODE REVIEW

**Comment by kgevorkyan** [comment]
The standard mode won't work until https://github.com/diktat-static-analysis/save-cloud/issues/333

However, I remove manually the extra command line args for save-cli and it worked fine, both for standard mode and rerun of standard mode

File: save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt [hunk 1]
```diff
@@ -114,15 +120,24 @@ class TestExecutionService(private val testExecutionRepository: TestExecutionRep
...
+                it.filePath
+            }
+            val testExecDto = it.copy(filePath = filePathWithDroppedPrefixForStandardMode)
             val foundTestExec = testExecutionRepository.findByExecutionIdAndTestPluginNameAndTestFilePath(
                 executionId,
                 testExecDto.pluginName,
                 testExecDto.filePath
             )
+            var testExecutionId: Long? = null
+            foundTestExec.ifPresent { testExecutionId = it.id }
```
Comment (petertrr) [inline save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt:140]:
```suggestion
            val testExecutionId: Long? = foundTestExec.map { testExecutionId = it.id }.orElse(null)
```

File: save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt [hunk 2]
```diff
@@ -114,15 +120,24 @@ class TestExecutionService(private val testExecutionRepository: TestExecutionRep
...
             "Agent with containerId=[$agentContainerId] was not found in the DB"
         }
+
         val executionId = agent.execution.id!!
         val lostTests: MutableList<TestExecutionDto> = mutableListOf()
         val counters = Counters()
-        testExecutionsDtos.forEach { testExecDto ->
+        testExecutionsDtos.forEach {
+            val filePathWithDroppedPrefixForStandardMode = if (it.filePath.startsWith(PREFIX_FOR_SUITES_LOCATION_IN_STANDARD_MODE)) {
+                it.filePath.dropWhile { it != '/' }.drop(1)
```
Comment (petertrr) [inline save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt:129]: Please add some comments, that the first branch is for `STANDARD` mode and the secobd - for `GIT` mode

File: save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt [hunk 3]
```diff
@@ -175,10 +177,19 @@ class DockerService(private val configProperties: ConfigProperties) {
         val testSuitesDir = resourcesPath.resolve(standardTestSuiteDir)
         // copy corresponding standard test suites to resourcesRootPath dir
         copyTestSuitesToResourcesPath(testSuitesForDocker, testSuitesDir)
+
+        // move additional files, which were downloaded into the root dit to the execution dir for standard suites
+        execution.additionalFiles?.split(";")?.filter { it.isNotBlank() }?.forEach {
+            val additionalFilePath = Paths.get(resourcesPath.resolve(File(it).name).absolutePath)
```
Comment (petertrr) [inline save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt:183]: Do we always have additional files in the root of resourcesPath? Maybe, having `resolve(it)` will be more reliable?

File: save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt [hunk 4]
```diff
@@ -175,10 +177,19 @@ class DockerService(private val configProperties: ConfigProperties) {
...
+            val destination = Paths.get(testSuitesDir.absolutePath).resolve(File(it).name)
+            log.info("Move additional file $additionalFilePath into $destination")
+            Files.move(additionalFilePath, destination, StandardCopyOption.REPLACE_EXISTING)
```
Comment (petertrr) [inline save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt:186]: When are we copying those files in `resourcesPath`? Maybe, we can move them in the proper directory at that point already?

Comment (kgevorkyan) [reply save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt:186]: We download files into `resourcesPath` during `uploadBin`, and at those moment  we doesn't know anything about `standardTestSuiteDir`, so I decide to provide logic of movement here

File: save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt (hunk 3 above)
Comment (kgevorkyan) [reply save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt:183]: Actually, in `execution.additionalFiles` we have the full paths to the files from our storage `/home/cnb/files/...`, but we also know, that during `uploadBin` execution we have already this files in the `resourcesPath`, with the same `.name` as in `execution.additionalFiles`

File: save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt (hunk 1 above)
Comment (kgevorkyan) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt]: Fixed

File: save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt (hunk 2 above)
Comment (kgevorkyan) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt:131]: Added

File: save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt (hunk 3 above)
Comment (petertrr) [reply save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt]: OK

**Review by petertrr — APPROVED** [review]

**Pull request merged**